src/style.css

```diff
@@ -0,0 +1,167 @@
+@font-face {
+  font-family: "Avenir";
+  src: local("Avenir-Book"),
+    local("Avenir-Heavy"),
+    url(fonts/Avenir-Book.ttf) format("truetype"),
+    url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 300;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.big_Field {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.firstBar {
+  display: flex;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  vertical-align: middle;
+  padding-left: 62px;
+  font-family: inherit;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 70px;
+  color: #3d4e61;
+  background: #ffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: none 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.firstBar::placeholder {
+  font-family: inherit;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 300;
```

Comment:
400.

Reply:
According to the condition of the assignment:
default `font-weight` must be 300.